python/ray/data/aggregate.py

```diff
@@ -18,31 +18,55 @@
 
 @PublicAPI
 class AggregateFn:
-    """Defines an aggregate function in the accumulator style.
+    """
+    Defines how to perform a custom aggregation in Ray Data.
```

Comment:
FYI, generally google docstring style is to put this on the first line, as it was before